### PR TITLE
Data added from literature

### PR DESCRIPTION
**Pull request by dodedic**
Added data from several sources.
Pyhton script updated to include said sources

### CODE REVIEW

**Comment by michaelweinold** [comment]
@dodedic, I saw that my review comments had not been addressed. Please ping me when your PR is actually ready for re-review. Thanks 🫡

**Comment by dodedic** [comment]
@michaelweinold, do you mean the comments from last Wednesday in Ben's latest message [here](https://github.com/sustainableaviation/demandmap/issues/2)?
Or is there a review tab on this PR that I cannot seem to find right now?

**Comment by dodedic** [comment]
For me it's not showing any review conversation nor comments on the previous commit (from two days ago). If I'm looking in the wrong place please let me know because I am slightly baffled right now.

image

**Review by michaelweinold — CHANGES_REQUESTED** [review]
...ok, maybe you'll see my comments now.

File: figures/forecasts_literature/forecasts_literature.py [hunk 1]
```diff
@@ -68,38 +175,98 @@
...
 ax.plot(
     df_airbus['year'],
-    df_airbus['traffic [Mpax-km]'],
-    color = 'black',
-    linewidth = 1,
-    label = 'Airbus (2023)'
+    df_airbus['traffic [RPK]'],
+    color='black',
+    linewidth=1,
+    label='Airbus'
```
Comment (michaelweinold) [inline figures/forecasts_literature/forecasts_literature.py:192]: ...add forecast years to all labels, please.

**Comment by michaelweinold** [comment]
Thank you for integrating my comments. You can go back to the Flake8 formatting for function calls - and then I'll merge the PR.

**Comment by dodedic** [comment]
- Colorblind-friendly colours added from [these suggestions](https://gist.github.com/thriveth/8560036)
- Position/size of legend adjusted
- Axis renamed
- ATI weird datapoint removed and replaced by calculated value with growth rate from literature

I am currently still struggling with the implementation of the vertical line at 2024 using plt.axvline (line 253). Maybe you could have a quick look at that and point out what I'm doing wrong. I need to rush off to work now since I'm working overnight so I thought I'd commit what I have already done so far.

**Pull request merged**